app.rb

```diff
@@ -0,0 +1,153 @@
+require 'json'
+require './loading'
+require './books/book'
+require './books/label'
+require './games/game'
+require './games/author'
+require './music/genre'
+require './music/music_album'
+require './movies/movie'
+require './movies/source'
+require './console_helper'
+
+class App
+  def initialize
+    @books = []
+    @labels = []
+    @games = []
+    @authors = []
+    @music_albums = []
+    @genres = []
+    @movies = []
+    @sources = []
+    puts 'Loading data from local files...'.blue
+    pause
+    list = { books: @books, labels: @labels, games: @games, authors: @authors, music_albums: @music_albums,
+             genres: @genres, movies: @movies, sources: @sources }
+    @loading = Loading.new(list)
+    @loading.load_labels
+    @loading.load_authors
+    @loading.load_genres
+    @loading.load_sources
+    puts 'Data loaded succesfully!'.green
+    pause(1)
+  end
+
+  def list_all_books
+    if @books.length.zero?
+      puts 'There are no books in your catalog for the time being. Maybe add one...?'
+    else
+      @books.each_with_index do |b, i|
+        puts "#{i + 1}) Publisher: #{b.publisher} | Publish date: #{b.publish_date} | Cover state: #{b.cover_state}"
+      end
+    end
+  end
+
+  def list_all_labels
+    if @labels.length.zero?
+      puts 'There are no labels in your catalog for the time being. Maybe add one...?'
+    else
+      @labels.each_with_index do |l, i|
+        puts "#{i + 1}) Title: #{l.title} | Color: #{l.color}"
+      end
+    end
+  end
+
+  def add_book(publisher, cover_state, publish_date, title, color)
+    @books << Book.new(publisher, cover_state, publish_date)
+    @labels << Label.new(title, color)
+    @labels.last.add_item(@books.last)
+    puts 'The provided book was successfully added!'.green
+    pause
+  end
+
+  def list_all_games
+    if @games.length.zero?
+      puts 'There are no games in your catalog for the time being. Maybe add one...?'
+    else
+      @games.each_with_index do |g, i|
+        puts "#{i + 1}) Multiplayer: #{g.multiplayer} | Publish date:
+         #{g.publish_date} | Last played: #{g.last_played_at}"
+      end
+    end
+  end
+
+  def list_all_authors
+    if @authors.length.zero?
+      puts 'There are no authors in your catalog for the time being. Maybe add one...?'
+    else
+      @authors.each_with_index do |a, i|
+        puts "#{i + 1}) First name: #{a.first_name} | Last name: #{a.last_name}"
+      end
+    end
+  end
+
+  def add_game(multiplayer, last_played_at, publish_date, first_name, last_name)
```

Comment:
The Game should have much more information than you are covering right now. Currently, you only have an `author` for the book and not the `label` or `genre` or `source`. Kindly add these accordingly. You can refer to [this video](https://www.loom.com/share/ae09042fe10c44e9906b8628314b49f5) where I went into detail about what you should fix/change.

Reply:
hello @muneebulrehman thanks for the review, but i would like you to check this link where is the project being assigned to each student [link](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md). Thanks